downstream/modules/platform/proc-gw-adjust-mapping-order.adoc

```diff
@@ -10,6 +10,17 @@ For example, if the first authenticator map is of type `is_superuser` and the tr
 
 And, if the second map is of type `is_superuser` and the trigger is based on the user having a specific group, any user logging in would initially be denied the `is_superuser` permission. However, any user with the specified group would subsequently be granted the `is_superuser` permission by the second rule. 
 
+The order of rules is important beyond whether you want to process organizations, teams or roles first. They can also be used to refine access and careful consideration is needed to avoid log in issues. 
```

Comment:
```suggestion
The order of rules is important beyond whether you want to process organizations, teams or roles first. They can also be used to refine access and careful consideration is needed to avoid log-in issues. 
```

Reply:
Fixed.